SearchHistory/tests.py

```diff
@@ -35,9 +27,8 @@ def test_del_search_history(self, searchHistory):
         searchHistory.delete()
         assert searchHistory not in SearchHistory.objects.all()
 
-    def test_get_5_last_professionals_search_by_client(self, persisted_search_history):
-        persisted_client_id = persisted_search_history[2]
-        expected_result = SearchHistory.objects.filter(client_id=persisted_client_id)
+    def test_get_5_last_professionals_search_by_client(self, persisted_search_history, client):
+        expected_result = SearchHistory.objects.filter(client_id=client)
         expected_result = expected_result.order_by('-date')[:5]
```

Comment:
Note that you assume here that expected_result size is bigger than 5...
What happens in the case that client has not yet searched for 5 professionals? (The size of  expected_result is smaller than 5)

Reply:
Also, what happens if client search for the same professional several times? 
Will we receive the same professional several times in the returned list?
Consider using distinct()